### ISSUE TITLE
Feil: Punktlistetegnene i uordnede lister leses opp av skjermlesere

### ISSUE THREAD

**Issue by piofinn** [fremtind/jokul#650]
**Feilbeskrivelse**
Punktene i punktlister er et UTF-tegn satt som `content` på `:before`-elementet til listepunktet. De leses dermed opp av skjermleseren (som "black diamond").

**For å reprodusere**
1. Gå til https://fremtind.github.io/jokul/list-react/documentation/List/
2. Start skjermleseren din
3. Vent til den unummererte listen leses opp
4. ~~Se~~ Hør feilen

**Forventet oppførsel**
Punktmerkingen bør ikke leses opp, kun selve innholdet i punktet.

**Mer info**
Mac med VoiceOver, testet in Chrome
